### DISCUSSION TITLE
Xamarin SignaturePad Migration to Maui

### DISCUSSION THREAD

**Discussion by robinhartauer** [CommunityToolkit/Maui#270] [Q&A] [answered]
Hi,
I want to ask if there are any plans to migrate the Xamarin.SignaturePad to Xamarin Maui or the Maui.CommunityToolkit?

The Repository https://github.com/xamarin/SignaturePad, has been archived so I'm not able to ask the question there. 

@jfversluis Can you help me with this question please?

Thank you for your help!
Greetings Robin

**Comment by jfversluis** [comment, marked as answer]
Hey there! We already have DrawingView which basically does the same thing. Did you see that? Would that be an option?

**Reply by VladislavAntonyuk** [reply]
https://github.com/CommunityToolkit/Maui/issues/108
we had some discussion about it. I investigated a new MAUI Graphics and left a comment how the new DrawingView can be implemented

**Reply by robinhartauer** [reply]
@jfversluis I didn't recognize that the DrawingView is the replacement. Thanks for the hint. I think we start to investigate the drawing View. 

Thanks for your quick help!

**Reply by VladislavAntonyuk** [reply]
Some time ago I created a small post on how to set it up https://vladislavantonyuk.azurewebsites.net/articles/Drawing-View-in-Xamarin-Community-Toolkit

**Reply by Yorch7** [reply]
hi!, is there a signaturepad or drawingview I can use on maui that doesn't require xamarin.forms to be installed?

**Reply by TheCodeTraveler** [reply]
Yes! @VladislavAntonyuk is currently working on implementing DrawingVIew for the .NET MAUI Community Toolkit: https://github.com/CommunityToolkit/Maui/issues/108

You can follow his progress here: https://github.com/CommunityToolkit/Maui/tree/drawingView

**Reply by amirvenus** [reply]
I am getting 404. Could you please double-check the link?

**Reply by VladislavAntonyuk** [reply]
try again

**Reply by amirvenus** [reply]
Will I be able to use it inside an android only project targetting net8.0-android (not maui)?

**Reply by VladislavAntonyuk** [reply]
yes

**Comment by CliffAgius** [comment]
Closed as answered
